composedtaskrunner-task-app-dependencies/pom.xml

```diff
@@ -11,13 +11,13 @@
 	<parent>
 		<artifactId>spring-cloud-dependencies-parent</artifactId>
 		<groupId>org.springframework.cloud</groupId>
-		<version>2.0.2.RELEASE</version>
+		<version>2.1.0.BUILD-SNAPSHOT</version>
```

Comment:
I assume this is the parent dependency we'll have for all the task app starters.

Reply:
We should move this way.  Yes.   Not only did I add the additional properties features, I updated the tests and did some small corrections to the documentation.